_dasldemos/fh-cbio-intro.md

```diff
@@ -0,0 +1,247 @@
+5. Zip your study folder before moving it into the `fh-dasl-cbio` S3 bucket. On a Mac, right-click on your study directory and click "Compress". On Windows, right-click on your study directory, select "Send to", then "Compressed (zipped) folder". If you prefer to use the command line, you can zip the folder using this command:
+    ```
+    # Go to the directory where your study folder is present
+    cd /path/to/directory/cancer_study_indentifier
+
+    # Zip the folder recursively
+    zip -r cancer_study_identifier.zip .
+    ```
+6. Transfer your data onto the `fh-dasl-cbio` S3 bucket. You can do that one of these 3 ways:
+    - **Motuz**:
+        1. Go to [Motuz](https://motuz.fredhutch.org/login)
+        2. Login using your Fred Hutch credentials
+        3. Follow the instructions [here](/compdemos/motuz/) to setup the `fh-dasl-cbio` bucket. 
+        4. Click the blue arrow and it will upload luad_berger_2024 (example) to the fh-dasl-cbio S3 bucket
+        ![cbio_15_motuz_upload](/dasldemos/assets/cbio_15_motuz_upload.png)
+    - **Mountain Duck:**
+        1. Follow [these](/compdemos/Mountain-CyberDuck/) instructions to setup Mountain Duck.
+        2. Then setup the `fh-dasl-cbio` S3 bucket by following [these](/compdemos/Mountain-CyberDuck/#configuring-mountain-duck) steps.
+        3. Open the `fh-dasl-cbio` bucket in finder.
+        ![mountain_duck_upload](/dasldemos/assets/cbio_16_mountain_duck_upload.png)
+        4. Copy over your study folder:
+    		1. Right-click on the study folder (wherever you have made it)
+    		2. Select Copy
+    		3. Got to the `fh-dasl-cbio` tab or window
+    		4. Right-click and click on Paste
+    - **Command line:**
+        Go to terminal and enter the following command to upload your folder into the `fh-dasl-cbio` S3 bucket:
+        ```
+        aws s3 cp /path/to/directory/cancer_study_identifier.zip s3://fh-dasl-cbio/
+        ```
+
+7. Our Airflow automation scripts will take care of the rest of the upload process from there and send you an email notification with details about the outcome. If upload was unsuccessful, the email will contain a detailed HTML file called a "validation report" that identifies which parts of your study are causing the issue and need updating. *Note:* If you do not receive an email notification indicating the success/failure of your study upload within an hour, reach out to the [cBioPortal Team](mailto:tfirman@fredhutch.org) for help identifying the issue.
+
+8. Go have fun and explore your data [on the Fred Hutch instance of cBioPortal!](https://cbioportal.fredhutch.org/)
+
+9. If you would like to update your study data or add more subjects to the cohort in the future, simply update your study data locally and reupload it to `fh-dasl-cbio`. Any subsequent uploads of the same study will overwrite the previous study data.
```

Comment:
> reupload it to `fh-dasl-cbio`

specify s3 bucket?

Reply:
Updated to specify that `fh-dasl-cbio` is an S3 bucket and step 6 instructions should be used for upload.